djangocms_rest_api/serializers/base.py

```diff
@@ -8,12 +8,27 @@
 from django.core.urlresolvers import reverse
 from rest_framework import serializers
 from rest_framework.serializers import ListSerializer
-from djangocms_rest_api.serializers.utils import RequestSerializer
+
+from djangocms_rest_api.serializers.utils import RequestSerializer, ClassLookupDict
 from djangocms_rest_api.serializers.mapping import serializer_class_mapping
+from djangocms_rest_api.serializers.fields import RecursiveField
 
 serializer_cache = {}
 
 
+class MenuSerializer(RequestSerializer, serializers.Serializer):
+    title = serializers.CharField(source='get_menu_title', read_only=True)
+    id = serializers.CharField(read_only=True)
```

Comment:
It is a representation for object field, to not make a confusion it is better to keep the name of actual object